src/Internal.AspNetCore.BuildTools.Tasks/Internal.AspNetCore.BuildTools.Tasks.csproj

```diff
@@ -1,42 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <Import Project="..\..\build\common.props" />
+  <!-- When possibly, make modifications to Shared.props to ensure changes made to this project are picked up by BuildTool.Tasks. -->
```

Comment:
What does this mean? `possibly` -> `possible` might help but are you talking about future changes?